Source/WebCore/css/process-css-pseudo-selectors.py

```diff
@@ -51,12 +51,23 @@
     })
 }
 
+# These count the number of -webkit- prefixes (excluding aliases).
+# Do not increase these numbers and instead consider using:
+# - a pseudo on standards track for things that are meant to be styled by arbitrary web content
+# - an `-internal-` prefix for things only meant to be styled in the user agent stylesheet
+# - an `-apple-` prefix only exposed to certain Apple clients (there is a settings-flag requirement for using this prefix)
+WEBKIT_PREFIX_COUNTS_DO_NOT_INCREASE = {
```

Comment:
I think an allow list would be better than a count.

Reply:
If we do that we should also allow list the single-colon pseudo-elements.

Reply:
I guess I could make that work, it will be a long allow list though

Reply:
Actually, thinking more I think the count might be better for reviewers, since it's less obvious in a long Github diff that you're adding something to an allow-list you shouldn't be adding to.

Reply:
I would tend to agree an allowlist is better if the number is small (e.g. under 10) but that's not currently the case. If the number is 60, adding an item to the allowlist will look like a normal change in a Github diff.